dash-pipeline/dockerfiles/Dockerfile.saithrift-client

```diff
@@ -28,6 +28,12 @@ RUN cd /saithrift-0.9 && \
     cd /SAI/test/ptf && \
     sudo python3 setup.py install
 
+# Copy and install dash pipeline utils
+ADD dash-pipeline/utils/ /dash-pipeline-utils
+RUN cd /dash-pipeline-utils && \
+    sudo python3 setup.py bdist_wheel && \
+    sudo pip install dist/dash_pipeline_utils*.whl
```

Comment:
do we need to update sonic-vs, so we can use in KVM / mgmt tests?

Reply:
Yes, will raise PR in repo sonic-buildimage to install this package in docker sonic-mgmt